src/cpu/x64/matmul/brgemm_matmul_utils.cpp

```diff
@@ -1824,8 +1847,16 @@ void init_aux_values(brgemm_matmul_conf_t &bgmmc,
                                           wei_stride / factor)
                 * factor;
     } else if (bgmmc.transposed_B) {
-        bgmmc.copy_B_wei_stride
-                = (wei_d.strides()[bgmmc.ndims - 1] * bgmmc.b_dt_sz);
+        if (wei_d.matches_one_of_tag(abcd) == format_tag::undef) {
```

Comment:
I would really switch `if`-`else` conditions in a way that "if abcd is matched, do what's now in else part", the other way around is less evident from readability perspective.

Besides that, maybe stop relying on those tags? It seems enough to check if the last stride is the minimal one. The condition for it to be dense is under question, though. Right now it's not required to be dense.